doc/source/index.rst

```diff
@@ -63,15 +69,28 @@ Here are some key features of PyDPF Composites:
 * Postprocessing of homogeneous elements.
 
 
+.. _limitations:
+
 Limitations
 '''''''''''
-- Layered elements (section data) that have not been preprocessed with ACP are not supported.
-  For information on converting legacy models, see `Import of Legacy Mechanical APDL Composite Models`_
-  in the Ansys Help.
 - Only the Mechanical APDL solver is supported.
-
+- The following operators and features are only supported if the model was preprocessed with ACP
+  and if the corresponding lay-up definition file is passed to the :class:`.CompositeModel`:
+  - The evaluation of the failure criteria for sandwich (:class:`FaceSheetWrinklingCriterion`, :class:`ShearCrimpingCriterion`)
+    for solid elements. Layered shell elements are always supported.
+  - The computation of interlaminar normal stresses (s3) for layered shell elements.
+    Otherwise, s3 is zero. This also affects 3D failure criteria which use s3, such as Puck 3D.
+    This limitation does not affect (layered) solid elements.
+  - The support of variable materials and material fields. Without ACP, only the
+    temperature is considered for the evaluation of variable material properties.
+  - Global plies, ply-wise scoping, and ply-wise failure criteria. Layer-wise post-processing is
```

Comment:
I disagree with Vale. Do hyphenate "ply-wise" and add it to Vale's accept.txt file.